src/IndexedDbProvider.ts

```diff
@@ -328,27 +329,33 @@ class IndexedDbTransaction implements NoSqlProvider.DbTransaction {
     }
 }
 
+function filterFullTextMetadataAndReturn<T>(schema: NoSqlProvider.StoreSchema, val: T): T {
+    if (val) {
+        _.each(schema.indexes, index => {
+            if (index.fullText) {
+                delete val['nsp_i_' + index.name];
```

Comment:
Question: Should the 'nsp_i_' constant be defined somewhere (or have a function do this)? Not sure how much I care :/